sql/npc_list.sql

```diff
@@ -23470,26 +23470,26 @@ INSERT INTO `npc_list` VALUES (17629748,'qm7','???',115,-506.764,17.375,-420.372
 INSERT INTO `npc_list` VALUES (17629751,'Goblin_Geologist','Goblin Geologist',128,-737.000,-10.125,-550.000,1,40,40,0,0,0,0,3,0x0000550000000000000000000000000000000000,0,NULL,1);
-INSERT INTO `npc_list` VALUES (17629752,'qm8','???',0,-345.000,0.500,826.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629753,'qm9','???',0,76.000,0.020,-27.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629754,'qm10','???',0,-60.000,1.278,-169.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629755,'qm11','???',0,-181.000,7.625,-107.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629756,'qm12','???',0,816.000,1.502,-568.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629757,'qm13','???',0,548.000,17.314,-946.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629758,'qm14','???',0,272.000,1.962,24.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629759,'qm15','???',0,101.000,0.000,-100.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629760,'qm16','???',0,-21.000,0.000,-18.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629761,'qm17','???',0,-211.000,10.341,-129.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629762,'qm18','???',0,-108.000,1.053,-170.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629763,'qm19','???',0,-142.000,1.252,-271.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629764,'qm20','???',0,-65.000,1.413,-270.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629765,'qm21','???',0,631.000,0.187,-419.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629766,'qm22','???',0,862.000,-8.500,-415.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629767,'qm23','???',0,613.000,8.981,-537.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17629768,'qm24','???',0,770.000,0.460,-668.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17629752,'qmAMK','qm8',0,-345.000,0.500,826.000,1,40,40,0,0,32,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
```

Comment:
I forget what if the polutils name is used for anything, but typically this isn't a thing we change. The first `name` field is the one we use to do internal lookups and fire off scripts etc. The second one is pulled from POLUtils (I assume, from the name?) and may or may not be what's send to the client. On other entity types it is anyway.

Reply:
Ya, I was trying to preserve the identity of the qm in the db somehow, as the first name field is all going to be identical.  Not sure of another way to do it.